README.md

```diff
@@ -1 +1,43 @@
-# contributor-website-v2
\ No newline at end of file
+# contributor-website-v2
```

Comment:
Give a more user-friendly title here. `Contributor Website` is just fine. I was having a different name for the same: `Community Website` since we already have a contributor website in existence and making a v2 on a separate repository would be counter-intuitive.

Reply:
I think `Contributor Website` will be better for two reasons :- 
1. That is what the domain name will be.
2. We'll archive the old contributor website repo once this is published.

I'll change the heading. Thanks!